### DISCUSSION TITLE
Is there any way to add a co-pilot button to Windows 10 style?

### DISCUSSION THREAD

**Discussion by cheonmux** [valinet/ExplorerPatcher#3049] [Q&A] [answered]
hello.

I'm using the Windows 10 style taskbar, but I'd like to add the CoPilot button that's in the 11 style.

Can someone please tell me how to do this? thank you!

**Comment by pyrates999** [comment, marked as answer]
duplicate of: https://github.com/valinet/ExplorerPatcher/discussions/2948

Please close this.
